### ISSUE TITLE
Failed test: BytesTest.testCopyToWithUnicode

### ISSUE THREAD

**Issue by ctubbsii** [apache/fluo#856]
Recently fixed Jenkins builds (was pointing to old git repo, prior to move to gitbox) and saw a [failed test][1] for `org.apache.fluo.api.data.BytesTest.testCopyToWithUnicode`:

```
Error Message

expected:<15> but was:<10>
Stacktrace

java.lang.AssertionError: expected:<15> but was:<10>
	at org.apache.fluo.api.data.BytesTest.testCopyToWithUnicode(BytesTest.java:339)
```

[1]: https://builds.apache.org/view/F/view/Fluo/job/Fluo/org.apache.fluo$fluo-api/118/testReport/junit/org.apache.fluo.api.data/BytesTest/testCopyToWithUnicode/

**Comment by cjmctague** [comment]
Interesting. Which encoding is Jenkins using? The test called `.getBytes()` which is using the default system encoding. The result of `15` is assuming the encoding is `UTF-8` I believe. Locally I'm getting the result of `15` and the only way I can produce a result of `10` is by using the encoding `.getBytes("CP1252")`

**Comment by ctubbsii** [comment]
CP1252 appears to be the default for Windows. Jenkins may have some builders which run on Windows.

**Comment by ctubbsii** [comment]
Test comment. Please ignore.

**Comment by ctubbsii** [comment]
Our Jenkins jobs are configured to run on "Hadoop||ubuntu" nodes, which should all be Linux, so I'm really not sure why this is happening. Either way, the test could be fixed so that line 334 call to `getBytes()` passes in `StandardCharsets.UTF_8`.

**Comment by cjmctague** [comment]
```
Error Message

expected:<c[†𝔊]e> but was:<c[â ð]e>
Stacktrace

org.junit.ComparisonFailure: expected:<c[†𝔊]e> but was:<c[â ð]e>
	at org.apache.fluo.api.data.BytesTest.testCopyToWithUnicode(BytesTest.java:343)
```
Broke another test with the fix in 3302fd6
